src/ui/public/styles/variables/colors.less

```diff
@@ -17,6 +17,8 @@
 @globalColorTeal: #00A69B;
 @globalColorOrange: #E5830E;
 @globalColorPurple: #7800A6;
+@globalColorSlightHue: #909AA1;
+@globalColorFocus: #e6f2f6;
```

Comment:
I see these two variables added in this PR but I'm not seeing them used anywhere in it. Can they be removed from this PR?

Reply:
Feel free to remove. I was using them when I was playing around with the styling. These are essentially my fake EUI variables in LESS. Totally safe to remove though.

Reply:
I deleted them 💀